### ISSUE TITLE
Добавить визуальные схемы к WP 

### ISSUE THREAD

**Issue by Eduard59** [SerendipityLab/CFSP#1]
1/ Из определения общей темы сообщества, мы определим критическую массу основателей необходимую для прохождения начальной точки роста исходного сообщества. Лучший способ сделать это - описать схему и отобразить экосистему. 
- Карта экосистемы идентифицирует все заинтересованные стороны в конкретном сообществе, на ней мы сможем отобразить их взаимозависимости, такая карта наиболее точно даст понять кто ключевые партнеры и сервисы, что нужно для первоначального старта создания минмимального исходного сообщества  так же это поможет определить размеры и объемы конкретного рынка. 
2/ Нарисовать простейшую и желаемую схемы оборота токена(токенов) платформа + биржа  
Далее.....

**Comment by alexeykrol** [comment]
Согласен, это следующий этап, пока структурирую. Сам текст меняется. Вчера внес изменения. Спасибо.
